design/20190708.certificate-request-crd.md

```diff
@@ -210,6 +408,38 @@ non-networking or other non-transient issue then the `Order` is marked as failed
 further processing will take place by the `CertificateRequest` controller on
 this resource.
 
+#### Issuing Controller
+
+Since external issuers have been built before the addition of Approved and
+Denied conditions, the issuing controller needs to be permissive. An external
+issuer may not honour an Approved condition and will sign and set a
+CertificateRequest as being Ready, before the request has been approved. The
+issuing controller must mark issuance as being successful in this case. In
+practice, this means that the issuing controller is never concerned with
+Approved conditions.
+
+External issuers that do not honour Denied conditions will sign
+CertificateRequests, even if they have a Denied condition set. In this case, the
+issuing controller will successfully complete the issuance of the Certificate.
+
+External Issuers and internal issuer that honour the Denied condition will never
+sign CertificateRequests with the Denied condition set, and thus never set the
+Ready condition. In this case, the issuing controller will consider this
```

Comment:
By "issuing controller", do you refer to external issuers? or internal issuers? or both?

Reply:
Both

Reply:
I realize that I am confusing "issuing controller" with [issuing_controller.go](https://github.com/jetstack/cert-manager/blob/master/pkg/controller/certificates/issuing/issuing_controller.go). Could we rephrase to remove the ambiguity?

Also, I am not too sure about what "in this case" refers to. Does it refer to the fact that the issuer is honoring?

```suggestion
Ready condition. We say that an issuer is "honoring" the Approval API when the issuer considers this
```